io.github.dbchoco.Salawat.yml

```diff
@@ -0,0 +1,48 @@
+app-id: io.github.dbchoco.Salawat
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk17
+command: salawat
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+  - --talk-name=org.freedesktop.portal.FileChooser
```

Comment:
again, why do you need it?

Reply:
The app sometimes need a file chooser, so I though this would give it a native look?